### ISSUE TITLE
Luminus template options don't seem to work

### ISSUE THREAD

**Issue by seancorfield** [seancorfield/clj-new#58]
From Rowan on Slack:

"I'm still having issues with that command, here is what I entered along with the output:
```
$ clojure -X:new :template luminus :name Rowan/guestbook :output '"mywebapp"' :args '["+h2" "+http-kit"]'
Downloading: luminus/lein-template/maven-metadata.xml from clojars
Unrecognized options: +h2, +http-kit
Supported options are: +sqlite, +hoplon, +kee-frame, +immutant, +datomic, +site, +undertow, +h2, +auth-jwe, +reitit, +reagent, +jetty, +cljs, +graphql, +service, +sassc, +oauth, +swagger, +servlet, +auth, +war, +http-kit, +expanded, +cucumber, +aleph, +mongodb, +basic, +postgres, +boot, +mysql, +re-frame, +kibit, +shadow-cljs
```
I don't understand why it says those are unrecognised options and then clearly lists them underneath as supported options"

**Comment by seancorfield** [comment]
Works just fine for me so maybe this is a Windows quoting issue?
```
$ clojure -X:new :template luminus :name Rowan/guestbook :output '"mywebapp"' :args '["+h2" "+http-kit"]'
Downloading: luminus/lein-template/maven-metadata.xml from clojars
Generating a Luminus project.
```

**Comment by seancorfield** [comment]
This is addressed by #59 in 1.1.297
